crates/bevy_input/src/gamepad.rs

```diff
@@ -47,23 +47,23 @@ pub struct Gamepads {
 
 impl Gamepads {
     /// Returns `true` if the `gamepad` is connected.
-    pub fn contains(&self, gamepad: &Gamepad) -> bool {
-        self.gamepads.contains(gamepad)
+    pub fn contains(&self, gamepad: Gamepad) -> bool {
+        self.gamepads.contains(&gamepad)
     }
 
-    /// An iterator visiting all connected [`Gamepad`]s in arbitrary order.
-    pub fn iter(&self) -> impl Iterator<Item = &Gamepad> + '_ {
-        self.gamepads.iter()
+    /// Returns an iterator over registered [`Gamepad`]s in an arbitrary order.
+    pub fn iter(&self) -> impl Iterator<Item = Gamepad> + '_ {
+        self.gamepads.iter().copied()
     }
 
     /// Registers the `gamepad` marking it as connected.
```

Comment:
Let's be consistent and add a comma here too :)
```suggestion
    /// Registers the `gamepad`, marking it as connected.
```